### ISSUE TITLE
pubmed.R rentrez 414 error: request too large

### ISSUE THREAD

**Issue by chreman** [OpenKnowledgeMaps/Headstart#197]
pinging @sckott , example when trying to run the pubmed-test.R
```
> input_data = get_papers(query, params)
 Hide Traceback
 
 Rerun with Debug
 Error in entrez_check(response) : 
  HTTP failure 414, the request is too large. For large requests, try using web history as described in the rentrez tutorial 
5.
stop("HTTP failure 414, the request is too large. For large requests, try using web history as described in the rentrez tutorial") 
4.
entrez_check(response) 
3.
make_entrez_query("esearch", db = db, term = term, config = config, 
    retmode = retmode, usehistory = usehistory, ...) 
2.
rentrez::entrez_search(db = "pubmed", term = query, retmax = limit, 
    mindate = from, maxdate = to, sort = sortby, use_history = TRUE) at pubmed.R#53
1.
get_papers(query, params) 

```

**Comment by chreman** [comment]
can't find anything in the latest change notes that would indicate a relation https://www.ncbi.nlm.nih.gov/books/NBK179288/#chapter6.Release_Notes

**Comment by sckott** [comment]
trying to run this Chris, but recent `tm` package version has lost the function `readTabular` and apparently `DataframeSource` has replaced it

tried that replacement but looks like structure of the output has changed too

https://stackoverflow.com/questions/47604511/the-readtabular-function-is-gone-in-the-newest-version-of-tm-what-do-we-use-a

**Comment by chreman** [comment]
At the moment we're stuck with an older version of `tm`, as we did not have the chance to update to their new API. Currently we use tm version 0.6-2 `devtools::install_version("tm", version="0.6-2")`

**Comment by sckott** [comment]
ok

**Comment by chreman** [comment]
Also note that with the current code the error may be not reproducible - as the [hotfix](https://github.com/OpenKnowledgeMaps/Headstart/commit/ad942074ecf37ce4859bb3b4e201de197f84cedc) is still in place. As this error refers to a too large request, maybe it may similarly be solved by your solution proposed for #257

**Comment by sckott** [comment]
- try to reproduce error with article types from drop down list added back in
- try both old and new `tm` package versions

**Comment by chreman** [comment]
Fixed in #420
